Works around a segfault when running with glibc-2.6, we need a proper solution later.
diff -rN -up old-pacman/src/pacman-g2/pacman-g2.c new-pacman/src/pacman-g2/pacman-g2.c
--- old-pacman/src/pacman-g2/pacman-g2.c	2007-05-19 14:32:16.000000000 +0200
+++ new-pacman/src/pacman-g2/pacman-g2.c	2007-05-19 14:32:16.000000000 +0200
@@ -202,7 +202,6 @@ static void cleanup(int signum)
 	}
 
 	/* free memory */
-	FREELISTPTR(pmc_syncs);
 	FREELIST(pm_targets);
 	FREECONF(config);
 

